index.bs

```diff
@@ -1612,7 +1612,7 @@ session.CapabilityRequest = {
   ? proxy: session.ProxyConfiguration,
   ? unhandledPromptBehavior: session.UserPromptHandler,
   Extensible
-};
+}
```

Comment:
I actually see some more semicolons across the file for CDDL definitions. Do those not cause trouble? Like the one for `browser.UserContext` and `network.CookieHeader`.

Reply:
Good catch, not sure why I missed those, fixed in 8ab6359.